conf.d/go-metro.yaml.example

```diff
@@ -0,0 +1,31 @@
+# NOTE: for go metro to run unprivileged, you will have to set CAP_NET_RAW capabilities on the binary:
+#        (debian based) $ sudo apt-get update libcap
+#        (redhat based) $ sudo yum install compat-libcap1
+#        (redhat based - alternatively) $ sudo yum install compat-libcap1
```

Comment:
Unrelated: let's add a comment to mention that `go-metro` logs to a separate file